mythril/laser/ethereum/transaction/transaction_models.py

```diff
@@ -178,11 +151,7 @@ def initial_global_state(self) -> GlobalState:
             self.code,
             calldata_type=self.call_data_type,
         )
-
-        global_state = GlobalState(self.world_state, environment, None)
-        global_state.environment.active_function_name = "constructor"
-
-        return global_state
+        return super().initial_global_state_from_environment(environment)
```

Comment:
what about active_function = "constructor"

Reply:
That's set as the default argument for `initial_global_state_from_environment`. I'll make it explicit, though for better readability